exercises/codec-server/practice/worker/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"log"
+
+	temporalconverters "github.com/temporalio/edu-converters-go-code/exercises/codec-server/solution/"
```

Comment:
Thanks for this -- I was a bit stumped by how to test this properly on a branch before merging, since the Go package manager has strange caching behavior when pulling from Github.